design/recovery-internals.md

```diff
@@ -52,10 +52,9 @@ Cluster controller (CC) decides if recovery should be triggered. In case the cur
 
 Recovery has 9 phases, which are defined as the 9 states in the source code: READING_CSTATE = 1, LOCKING_CSTATE = 2, RECRUITING = 3, RECOVERY_TRANSACTION = 4, WRITING_CSTATE = 5, ACCEPTING_COMMITS = 6, ALL_LOGS_RECRUITED = 7, STORAGE_RECOVERED = 8, FULLY_RECOVERED = 9.
 
-The recovery process is like a state machine, changing from one state to the next state. 
-We will describe in the rest of this document what each phase does to drive the recovery to the next state.
+The recovery process is like a state machine, changing from one state to the next state. `ClusterRecovery.actor` implements the cluster recovery. We will describe in the rest of this document what each phase does to drive the recovery to the next state.
 
-Recovery tracks the information of each recovery phase in `<RecoveryEventPrefix>RecoveryState` trace event. By checking the message, we can find which phase the recovery is stuck at. The status used in the `<RecoveryEventPrefix>RecoveryState` trace event is defined as `RecoveryStatus` structure in `RecoveryState.h`. The status, instead of the name of the 9 phases, is typically used in diagnosing production issues.
+The system tracks the information of each recovery phase via trace events. The `ServerKnob::CLUSTER_RECOVERY_EVENT_NAME_PREFIX` determines the event name prefix.  Recovery tracks the information of each recovery phase in `<Prefix>RecoveryState` trace event. By checking the message, we can find which phase the recovery is stuck at. The status used in the `<Prefix>RecoveryState` trace event is defined as `RecoveryStatus` structure in `RecoveryState.h`. The status, instead of the name of the 9 phases, is typically used in diagnosing production issues.
```

Comment:
Maybe explain here that the event name was changing from "MasterRecoveryState" to "ClusterRecoveryState". The prefix know is to given an option for old tooling to work.

Reply:
Great point!
Shall make necessary changes, thanks for the suggestion.

Reply:
Just to confirm, in its current state, event names are NOT changed; default value for the KNOB is "Master"; but worth recording this in the documentation for better clarity. 
Thanks